tests/cases/fourslash/refactorAddOrRemoveBracesToArrowFunction9.ts

```diff
@@ -0,0 +1,13 @@
+/// <reference path='fourslash.ts' />
+
+//// const foo = /*a*/a/*b*/ => (1, 2, 3);
```

Comment:
All of these tests seem to use the same selection range.  What if the parameter has more than one character and only part of it is selected?  What if a different part of the arrow function is selected?

Reply:
Does it matter if the selection is empty?